tools/custom-organization-policy-library/build/custom-constraints/gke/gkeRequireGKEMetadataServer.yaml

```diff
@@ -3,14 +3,13 @@
 
 #@ if constraint.to_generate():
 name: #@ constraint.constraint_name()
-resource_types: 
-- container.googleapis.com/Cluster
-- container.googleapis.com/NodePool 
-condition: 1 == 0
-action_type: DENY
-method_types: 
+resourceTypes:
+- container.googleapis.com/NodePool
+methodTypes:
 - CREATE
-- UPDATE 
+- UPDATE
+condition: resource.config.workloadMetadataConfig.mode != 'GKE_METADATA'
```

Comment:
Confirmation we need to use uppercase, screenshot ?

Reply:
![image](https://github.com/vannicktrinquier/professional-services/assets/162398098/1a499d79-daa0-4809-9acb-0ace67050d14)
not sure if uppercase is needed. did not test if lower case works

Reply:
Oki i checked here
https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/NodeConfig#mode

GKE_METADAYA is good